chapter11/mdw/Island.pm

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/perl
+use utf8;
+use strict;
+use warnings;
+
+use v5.12;
+
+package Island 0.11;
+
+#Translations
+our @day = qw(ark dip wap sen pop sep kir);
+our @month = qw(diz pod bod rod sip wax lin sen kun fiz nap dep);
+
+#This method turns the number of the day into its name
+sub number_to_day_name { 
+	my ( $num ) = @_;
+	if ( $num && ( $num >= 0 ) && ( $num < 7 ) ){
```

Comment:
Also, `if ( $num )` will be false for `0` right? So this also breaks on Sundays.